RetrospectiveExtension.Frontend/components/feedbackBoardContainer.tsx

```diff
@@ -1352,10 +1353,11 @@ console.log(createdBoard);
                               text="Reconnect" />
                             <IconButton
```

Comment:
I can dig in more, but does this IconButton need any screen reader text?

Reply:
it's an incredibly good suggestion, honestly, I don't know. let me check.

Reply:
it seems it's possible, here is the official; documentation; [IButtonProps interface Implementation](https://developer.microsoft.com/en-us/fluentui#/controls/web/button#implementation)

I'd like to have a separate issue to check the all the codebase in terms of accessibility, like, missing `aria-label` attribute, etc.

at the moment I'm marking this comment as resolved, but I'll create an issue item to track

Reply:
Agree that it would be out of scope for this PR, thank you for the clarification!